src/components/NoteDataProvider.tsx

```diff
@@ -23,25 +32,36 @@ interface NoteDataProviderProps {
   physician: string;
 }
 
-export const NoteContext = createContext<Note | null>(null);
+interface NoteContextProps {
+  note: Note | null;
+  setNote: Dispatch<SetStateAction<Note>> | (() => void);
+}
+
+export const NoteContext = createContext<NoteContextProps>({
+  note: null,
+  setNote: () => {
+    // do nothing.
+  }
+});
 
 export const NoteDataProvider: FC<NoteDataProviderProps> = ({ children, date, physician }) => {
```

Comment:
Now that the note below uses the UserProvider, can the physician prob be removed?

Reply:
Good call 👍